examples/basic_vault_transfer.py

```diff
@@ -0,0 +1,17 @@
+from hyperliquid.utils import constants
+import example_utils
+
+
+def main():
+    address, info, exchange = example_utils.setup(constants.TESTNET_API_URL, skip_ws=True)
+
+    if exchange.account_address != exchange.wallet.address:
+        raise Exception("Agents do not have permission to perform internal transfers")
+
+    # Transfer 5 usd to the HLP Vault for demonstration purposes
+    transfer_result = exchange.vault_usd_transfer("0xa15099a30bbf2e68942d6f4c43d70d04faeab0a0", True, 5000000)
```

Comment:
nit
```suggestion
    transfer_result = exchange.vault_usd_transfer("0xa15099a30bbf2e68942d6f4c43d70d04faeab0a0", True, 5_000_000)
```

Reply:
Also, can you select the hlp address based on if exchange is mainnet vs. testnet (or add a comment to change it if using a different environment)